src/phy.cpp

```diff
@@ -98,7 +98,7 @@ namespace PhyParser {
     std::vector<uint16_t> indices;
     indices.reserve(ledge.trianglesCount * 3);
 
-    uint16_t maxVertexIndex = 0;
+    unsigned int maxVertexIndex = 0;
```

Comment:
Why was this changed?

Reply:
To avoid narrowing errors in the proceeding `std::max` call below this line